src/main.rs

```diff
@@ -249,8 +253,6 @@ enum Commands {
     },
 
     /// Append records to a stream.
-    ///
-    /// Currently, only newline delimited records are supported.
```

Comment:
The newline delimited is still true right :)

Reply:
Yeah, but the I am not very keen on having it documented for all formats. I do think we can have a format with multi-line records.

Reply:
Added the documentation for each format. Clap displays it separately.